test/selectors/channels.test.js

```diff
@@ -1084,12 +1084,7 @@ describe('Selectors.Channels', () => {
                 sidebarPrefs.favorite_at_top === 'true',
             );
             assert.notDeepEqual(fromModifiedState, fromRecencyInChan5State);
-
-            // first item should match channel2 since it has user mention
-            assert.ok(fromRecencyInChan5State[0].items[0] === channel2.id);
-
-            // second item should match chan5 since it's the most recent among other channels
-            assert.ok(fromRecencyInChan5State[0].items[1] === chan5.id);
+            assert.ok(fromRecencyInChan5State[0].items[0] === chan5.id);
```

Comment:
Reverted to the one it was before - https://github.com/mattermost/mattermost-redux/pull/719/files#diff-690b880bab24707f9175df8ffd2f28efL1083